modules/op-release-notes-1-0.adoc

```diff
@@ -0,0 +1,88 @@
+
+[id="known-issues-1-4-0_{context}"]
+== Known issues
+* If you are upgrading from an older version of {pipelines-title}, you will need to delete your existing deployments before upgrading to {pipelines-title} version 1.0.
+* Submitting the same `v1alpha1` Tasks more than once will result in an error. Use `oc replace` instead of `oc apply` when re-submitting a `v1alpha1` Task.
+
+[id="fixed-issues-1-0_{context}"]
+== Red Hat Fixed issues
+* link:https://issues.redhat.com/browse/SRVKP-637[SRVKP-637] : Previously, the `DeploymentConfig` Task triggered a new deployment build even when an image build was already in progress. This caused the deployment in Pipeline to fail. With this fix, the `deploy task` command  is now replaced with the `oc rollout status` command which waits for the in-progress deployment to finish.
+* link:https://issues.redhat.com/browse/SRVKP-631[SRVKP-631]: Support for `APP_NAME` parameter is now added in Pipeline templates.
+* link:https://issues.redhat.com/browse/SRVKP-616[SRVKP-616]: Pipeline template for Java S2I failed to look up image in the registry. With this fix, the look up will now be performed using the existing image Pipeline Resources instead of user provided `IMAGE_NAME` parameter.
+* link:https://issues.redhat.com/browse/SRVKP-383[SRVKP-383]: All the Openshift Pipelines images are now based on the Red Hat Universal Base Images (UBI).
+* link:https://issues.redhat.com/browse/SRVKP-700[SRVKP-700]:  Previously, `tkn version` command displayed the Pipeline version as `unknown` when the Pipeline is installed in the namespace other than `tekton-pipelines`. With this fix, `tkn version` command now displays the correct Pipeline version in any namespace.
+* link:https://issues.redhat.com/browse/SRVKP-698[SRVKP-698]: `tkn version -c` command is now modified to not show newer versions of the CLI available upstream.
+* link:https://issues.redhat.com/browse/SRVKP-695[SRVKP-695]: Non-admin user can now list ClusterTriggerBindings.
+* link:https://issues.redhat.com/browse/SRVKP-693[SRVKP-693]: EventListener CompareSecret function is now fixed for CEL Interceptor.
+* link:https://issues.redhat.com/browse/SRVKP-690[SRVKP-690]: The `list`, `describe` and `start` sub-commands for `task` and `clustertask` now correctly display the output in case a Task and ClusterTask have same names.
+* link:https://issues.redhat.com/browse/SRVKP-671[SRVKP-671]: Previously, OpenShift Pipelines Operator modified the privileged CSS, which caused a breaking error during cluster upgrade. This error is now fixed.
+* link:https://issues.redhat.com/browse/SRVKP-669[SRVKP-669]: In the `tekton-pipelines` namespace with configuration map, the timeouts of all the TaskRuns and PipelineRuns is now set to the value of `default-timeout-minutes`field.
```

Comment:
In the `tekton-pipelines` namespace, the timeouts of all the TaskRuns and PipelineRuns is now set to the value of `default-timeout-minutes`field using the configuration map. ?
Add space before field